### ISSUE TITLE
Error messages around user[data] and around ['data']['channel']

### ISSUE THREAD

**Issue by dorel14** [DandyDev/slack-machine#526]
Hello 
i migrate to lastest slack-machine and have a lot fo messages like these in my log file 
~~~
[2021-12-17 12:09:13][ERROR] slack_sdk.rtm_v2 __init__.py:run_message_listeners:338 | Failed to run a message listener: 'data'
Traceback (most recent call last):
  File "/home/supportbot/bot/lib/python3.6/site-packages/slack_sdk/rtm_v2/__init__.py", line 336, in run_message_listeners
    listener(self, message)
  File "/home/supportbot/bot/lib/python3.6/site-packages/slack_sdk/rtm_v2/__init__.py", line 172, in new_message_listener
    func(_self, event)
  File "/home/supportbot/bot/lib/python3.6/site-packages/machine/clients/singletons/slack.py", line 147, in <lambda>
    self.rtm_client.on('user_change')(lambda client, event: self._on_user_change(client, event))
  File "/home/supportbot/bot/lib/python3.6/site-packages/machine/clients/singletons/slack.py", line 82, in _on_user_change
    user = self._register_user(payload['data']['user'])
~~~
Is this coming from one of my plugins ?

**Comment by repudi8or** [comment]
in machine/clients/singletons/slack.py, the payload data structure seems to have changed... what was (payload['data']['user']) is now (payload['user']) .... at least for _on_user_change

**Comment by DandyDev** [comment]
I will have a look at it. Is there any action in Slack that guarantees triggering this error, so I can reproduce?

**Comment by DandyDev** [comment]
Also @dorel14 please use triple back-ticks next time so your stack trace is readable :)

```
Now you can see
Each line
Separately
```

**Comment by dorel14** [comment]
Here it is :)

I think it is each time slack machine reconnect to slack

**Comment by dorel14** [comment]
Hi also have this kind of message when creating a new channel 

~~~
[2022-02-15 09:20:34][ERROR] slack_sdk.rtm_v2 __init__.py:run_message_listeners:338 | Failed to run a message listener: 'data'
Traceback (most recent call last):
  File "/home/devsupportbot/lib/python3.6/site-packages/slack_sdk/rtm_v2/__init__.py", line 336, in run_message_listeners
    listener(self, message)
  File "/home/devsupportbot/lib/python3.6/site-packages/slack_sdk/rtm_v2/__init__.py", line 172, in new_message_listener
    func(_self, event)
  File "/home/devsupportbot/lib/python3.6/site-packages/machine/clients/singletons/slack.py", line 126, in <lambda>
    lambda client, event: self._on_channel_created(client, event))
  File "/home/devsupportbot/lib/python3.6/site-packages/machine/clients/singletons/slack.py", line 86, in _on_channel_created
    channel_resp = self.web_client.conversations_info(channel=payload['data']['channel']['id'])
KeyError: 'data'
~~~

**Comment by DandyDev** [comment]
I fixed this in [v0.25.0](https://pypi.org/project/slack-machine/0.25.0/). Please upgrade
